packages/api/deploy/gnomad-api-controller.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         - name: GNOMAD_MONGO_URL
           value: mongodb://gnomad-mongo:27017/exac
         - name: ELASTICSEARCH_URL
-          value: 10.4.0.11:9200
+          value: 10.7.245.23:9200
```

Comment:
Similar to how `GNOMAD_MONGO_URL` uses the `gnomad-mongo` service name, can this use `elasticsearch` instead of a hard coded IP address that will differ on every cluster.